components/Filters/spamlist/SearchEmailIntoList.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { SearchInput, useApiWithoutResult } from 'react-components';
+import { getIncomingDefaults } from 'proton-shared/lib/api/incomingDefaults';
+import { noop, debounce } from 'proton-shared/lib/helpers/function';
+
+function SearchEmailIntoList({ className, onBeforeRequest, onAfterRequest }) {
```

Comment:
`onBeforeRequest` and `onAfterRequest` should be handle by the parent component to handle state change there.

```suggestion
function SearchEmailIntoList({ className, value, onChange }) {
```